examples/kubernetes.yml

```
@@ -38,6 +38,17 @@ spec:
           volumeMounts:
             - mountPath: /data
               name: data
+        livenessProbe:
+          exec:
+            command: 
+              - /bin/sh
+              - -c 
+              - '/mc-monitor status-bedrock --host 127.0.0.1 --port 19132 | grep online'
```

With easy-add this becomes
```suggestion
              - '/usr/local/bin/mc-monitor status-bedrock --host 127.0.0.1 --port 19132 | grep online'
```